news.php

```diff
@@ -1,20 +1,22 @@
-<!DOCTYPE html>
-<html>
+<!doctype html>
+<html lang="en">
   <head>
-    <link rel="stylesheet" href="news-styles.css" />
-  </head>
+    
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
 
-  <body>
-    <!--Header-->
-    <div id="header"></div>
-    <div id="searchLinks"></div>
-    <div id="header-inner"></div>
+    
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
 
-    <div id="container">
-      <div id="content-container">
-        <div id="content">
-        <p class="header-large">News</p>
+    <title>Behind the Reel | News</title>
+  </head>
+  
+  <?php include('nav-bar.php'); ?>
+  <body>
```

Comment:
the nav bar needs to be inside the body... right now as its written it will try to insert _between_ head and body

Reply:
Resolved w/ commit #938bc24a6